Examples/Framework/include/ActsExamples/Validation/DuplicationPlotTool.hpp

```diff
@@ -23,13 +23,13 @@ class TProfile;
 
 namespace ActsExamples {
 
-/// Tools to make duplication rate and duplication number plots to show tracking
```

Comment:
Why not doxygen-comments like `///`?

Reply:
ah looks like I messed up the merge conflict - will fix, thanks!